.github/workflows/on-push-release.yml

```diff
@@ -22,26 +22,3 @@ jobs:
       server_version: main
       ubuntu_version: ${{ matrix.ubuntu_version }}
       compiler: ${{ matrix.compiler }}
-
```

Comment:
These are largely redundant with the PR workflow, no real need to have them on push.